internal/metamorph/processor_options.go

```diff
@@ -69,6 +69,12 @@ func WithProcessStatusUpdatesInterval(d time.Duration) func(*Processor) {
 	}
 }
 
+func WithBulkProcessInterval(d time.Duration) func(*Processor) {
+	return func(p *Processor) {
+		p.bulkProcessInterval = d
+	}
+}
+
```

Comment:
I'm thinking if we're allowing to configure the interval, maybe we should also make the bulk size configurable, instead of having it always set to `200`?

Also, I was thinking that `WithProcessTransactionsInterval` would sound good as well, but it's always hard with naming 😄